Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,109 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = (
+      %w[brew core cask] +
+      OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") } +
+      OFFICIAL_CASK_TAPS
+    ).freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `contributions [email]`
+
+        Contributions to Homebrew repos for a user.
+      EOS
```

Comment:
I think this should be:

```suggestion
      usage_banner "`contributions` [<email>]"
      description <<~EOS
        Contributions to Homebrew repos for a user.
      EOS
```